scholarx/archive/functions.js

```diff
@@ -49,12 +60,47 @@ async function getData() {
 
 async function loadData() {
     const payload = await getData();
-    const mentorProfiles = payload.mentor2021.concat(payload.mentor2020).concat(payload.mentor2019);
+    [mentor[2019],mentor[2020],mentor[2021]] = [payload.mentor2019,payload.mentor2020,payload.mentor2021];
+    [mentee[2019],mentee[2020],mentee[2021]] = [payload.mentee2019,payload.mentee2020,payload.mentee2021];
+ 
+    const mentorProfiles = mentor[2019].concat(mentor[2020]).concat(mentor[2021]);
     let mentor_Profiles = Mustache.render($("#templateMentors").html(), { "mentorProfiles": mentorProfiles });
     $("#mentorProfiles").html(mentor_Profiles);
 
-    const menteeProfiles = payload.mentee2021.concat(payload.mentee2020).concat(payload.mentee2019);
+    const menteeProfiles = mentee[2019].concat(mentee[2020]).concat(mentee[2021]);
     let mentee_Profiles = Mustache.render($("#templateMentees").html(), { "menteeProfiles": menteeProfiles });
     $("#menteeProfiles").html(mentee_Profiles);
 }
 loadData();
+function renderProfiles(mentorYear,menteeYear) {
+    let mentor_Profiles = Mustache.render($("#templateMentors").html(), { "mentorProfiles": mentorYear });
```

Comment:
Use camel case for variable naming
```suggestion
    let mentorProfiles = Mustache.render($("#templateMentors").html(), { "mentorProfiles": mentorYear });
```